content/backend/graphql-js/7-subscriptions.md

```diff
@@ -247,19 +337,24 @@ mutation {
 
 </Instruction>
 
-Now observe the Playground where the subscription was running:
+Now observe the Playground where the subscription was
+running:
 
 ![](https://imgur.com/0BJQhWj.png)
 
 ### Adding a voting feature
 
 #### Implementing a `vote` mutation
 
-The next feature to be added is a voting feature which lets users _upvote_ certain links. The very first step here is to extend your Prisma data model to represent votes in the database.
+The next feature to be added is a voting feature which lets
+users _upvote_ certain links. The very first step here is to
+extend your Prisma data model to represent votes in the
+database.
 
 <Instruction>
 
-Open `prisma/schema.prisma` and adjust it to look as follows:
+Open `prisma/schema.prisma` and adjust it to look as
+follows:
 
 ```graphql{7,16,19-23}(path=".../hackernews-node/prisma/schema.prisma")
```

Comment:
Let's update the diffs in this section
```suggestion
```graphql{8,17,20-27}(path=".../hackernews-node/prisma/schema.prisma")
```